From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 12 Jul 2012 11:56:45 +0100
Subject: use CFLAGS

Don't specify -O2 and -g (on by default) so they can be overwritten
by builder, e.g., dpkg-buildflags.
---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index f5add4d..e71e7b5 100644
--- a/Makefile
+++ b/Makefile
@@ -8,7 +8,7 @@ PREFIX=$(prefix)
 
 # Override default compiler and flags
 CC=	gcc
-CFLAGS=	-g -Wall -ansi -pedantic -O2
+CFLAGS+=	-Wall -ansi -pedantic
 
 # You may try one of the following, if you get "wrong interpreter" errors
 #LDFLAGS+=	-Wl,-no_pie
@@ -75,7 +75,7 @@ all:	default s9e
 s9e:	s9e-core.image
 
 s9:	s9.o s9.h $(EXTRA_OBJS)
-	$(CC) -o s9 $(LDFLAGS) s9.o $(EXTRA_OBJS) $(EXTRA_LIBS)
+	$(CC) $(CFLAGS) -o s9 $(LDFLAGS) s9.o $(EXTRA_OBJS) $(EXTRA_LIBS)
 
 s9.o:	s9.c s9-real.c s9.h
 	$(CC) -o s9.o $(CFLAGS) $(DEFS) -c s9.c
